daemon/algod/api/algod.oas2.json

```diff
@@ -4087,22 +4090,6 @@
         }
       }
     },
-    "KvDelta": {
```

Comment:
Because unused/references?

Reply:
Yeah, it was just hanging out, doin' nothing.